plugins/outputs/iotdb/iotdb.go

```diff
@@ -30,9 +37,11 @@ type IoTDB struct {
 	ConvertUint64To string          `toml:"uint64_conversion"`
 	TimeStampUnit   string          `toml:"timestamp_precision"`
 	TreatTagsAs     string          `toml:"convert_tags_to"`
+	SanitizeTags    string          `toml:"sanitize_tag"`
 	Log             telegraf.Logger `toml:"-"`
 
-	session *client.Session
+	SanityRegex []*regexp.Regexp
```

Comment:
Please do not export this field as it is (and should be) only used internally
```suggestion
	sanityRegex []*regexp.Regexp
```